### PR TITLE
Add doppler config and update README

### PR DESCRIPTION
**Pull request by 0xemc**
## Description

Doppler config for `carbonmark-api`

### CODE REVIEW

**Comment by 0xemc** [comment]
@biwano if you have some time could you test this out by following the instructions in the `carbonmark-api/README.md`?

I also sent you an invite to doppler to your bruno@carbonmark.com email

File: carbonmark-api/README.md [hunk 1]
```diff
@@ -12,41 +12,32 @@
...
-\*\*Note all `vercel` commands should be run from within the root klimadao (monorepo) directory.
-
-\*\*All others should be run from within the `carbonmark-api` folder
+### Environment Variables
 
-### Link your local copy of the project with vercel
+Secrets are managed via Doppler and exposed as environment variables during run time. Therefore we need to setup the Doppler CLI
 
-Install the vercel CLI
+Install the doppler CLI
```
Comment (sprrwhwk) [inline carbonmark-api/README.md:23]:
```suggestion
[Install the doppler CLI](https://docs.doppler.com/docs/install-cli#installation)
```

**Review by sprrwhwk — CHANGES_REQUESTED** [review]
I would link to the installation docs instead. I know at least one team member who's using windows 😄

**Comment by biwano** [comment]
> @biwano if you have some time could you test this out by following the instructions in the `carbonmark-api/README.md`?
> 
> I also sent you an invite to doppler to your [bruno@carbonmark.com](mailto:bruno@carbonmark.com) email

@0xemc  It worked for me.

**Comment by 0xemc** [comment]
Converting to draft as we would need to provide everyone with doppler logins in order to locally develop

**Review by sprrwhwk — APPROVED** [review]

**Comment by 0xemc** [comment]
Closing this in favour of https://vercel.com/changelog/sensitive-environment-variables-are-now-available

**Pull request closed without merging**